plugins/obs-x264/obs-x264.c

```diff
@@ -205,6 +208,7 @@ static obs_properties_t *obs_x264_props(void *unused)
 			OBS_COMBO_TYPE_LIST, OBS_COMBO_FORMAT_STRING);
 	obs_property_list_add_string(list, TEXT_NONE, "");
 	add_strings(list, x264_tune_names);
+	obs_properties_add_bool(props, "dynamic_variable_bitrate", TEXT_DYN_BITRATE);
```

Comment:
Style:  Break lines to avoid exceeding 80 columns.  Suggestion:

```
	obs_properties_add_bool(props, "dynamic_variable_bitrate",
			TEXT_DYN_BITRATE);
```